src/index.js

```diff
@@ -2,18 +2,18 @@ import * as plugin from "./plugin";
 import CldContext from "./components/CldContext/CldContext";
 import CldImage from "./components/CldImage/CldImage";
 import CldVideo from "./components/CldVideo/CldVideo";
+import CldPoster from './components/CldVideo/CldPoster';
+import CldTransformation from './components/CldTransformation/CldTransformation';
 
-const components = {
-  CldContext,
-  CldImage,
-  CldVideo
+const allComponents = { CldContext, CldImage, CldVideo, CldTransformation, CldPoster };
+
+const Cloudinary = {
+  install: (Vue, options = {}) => {
+    plugin.install.call(plugin, Vue, options.components ? options : { ...options, components: allComponents })
+  }
 };
 
 export {
-  plugin,
-  plugin as default,
-  components,
-  CldContext,
-  CldImage,
-  CldVideo
+  Cloudinary as default,
```

Comment:
I'd suggest changing this to

```js
Cloudinary as default,
Cloudinary,
```

Just to avoid a bug reports when someone contrary to docs decides to import as

```js
import { Cloudinary, CldImage } from "cloudinary-vue";
```